inspirehep/modules/theme/templates/inspirehep_theme/accounts/settings/profile.html

```diff
@@ -18,60 +18,76 @@
+        {%- endfor %}
+      </ul>
+    </div>
+  </div>    
   </div>
-</div>
-<div class="row" id="settings-form">
-  <div class="col-sm-11 col-md-11">
-    <form method="POST" role="form">
-      {%- for field in profile_form -%}
-        {{ render_field(field) }}
-      {%- endfor -%}
-      <div class="form-actions">
-        <button type="submit" name="submit" value="profile" class="btn btn-success"><i class="fa fa-check"></i> {{ _("Update Profile") }}</button>
+  <div class="col-md-9">
+  <div class="panel panel-default custom-panel">
+    <div class="panel-heading"><i class="fa fa-user fa-fw"></i> <strong>Profile</strong></div>
+      <div class="panel-body">
+      {% if current_user.note == "2" and verification_form %}
+      <form method="POST" role="form">
+      {{ verification_form.csrf_token }}
+        <div class="alert alert-warning custom-alert">
+        {{ _("You have not yet verified your email address.") }} {{ verification_form.send_verification_email(class_="btn btn-default btn-xs") }}
+        </div>
+      <input type="hidden" name="submit" value="verification" />
+      </form>
+      {% endif %}
+      <div class="row"><div class="col-md-12"><p></p></div></div>
+      <div class="row" id="top-row">
+        <div class="col-lg-2" id="user-icon">
+          <img id="profile-picture" class="img-round" src="{{ current_user.email }}" data-toggle='tooltip' data-placement='left' title='Your profile picture can be set at gravatar.com'>
+        </div>
+        <div class="col-lg-10 profile-summary">
+          {%- if current_user.given_names or current_user.family_name %}
+            <h4 class="media-heading">{{ current_user.given_names ~ " " ~ current_user.family_name }}</h4>
+          {%- else %}
+            <h4 class="media-heading">{{ current_user.nickname }}</h4>
+          {%- endif %}
+          <p class="text-muted">{{ current_user.email }}</p>
+        </div>
+      </div>
+      <div class="row" id="settings-form">
+        <div class="col-sm-11 col-md-11">
+          <form method="POST" role="form">
+            {%- for field in profile_form -%}
+              {{ render_field(field) }}
+            {%- endfor -%}
+            <div class="form-actions">
+              <button type="submit" name="submit" value="profile" class="btn btn-success"><i class="fa fa-check"></i> {{ _("Update Profile") }}</button>
+            </div>
+          </form>
+        </div>
+        <div class="col-sm-1 col-md-1"></div>
+      </div>
+      <div class="row"><div class="col-md-12"><p></p></div></div>
       </div>
-    </form>
+    </div>
   </div>
-  <div class="col-sm-1 col-md-1"></div>
 </div>
-<div class="row"><div class="col-md-12"><p></p></div></div>
-{{ helpers.panel_end() }}
 {% endblock %}
 {% block javascript %}
+{{ super() }}
 <script type="text/javascript">
 require([
   'jquery',
   'bootstrap',
   ], function ($) {
-    {% if user.note == "1" %}
+    {% if current_user.note == "1" %}
```

Comment:
This note used to indicate that the user had verified her account, but does not exist any more in the model:

https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/models.py#L53

The model contains `confirmed_at` that I think should be filled in when the user confirms the email but at the moment the code to do that does not exist in Invenio.

So temporarily you can do `{% if current_user.confirmed_at %}` and we can see if add the functionality to Invenio ourselves.